lib/plugins/aws/customResources/resources/eventBridge/lib/utils.js

```diff
@@ -2,7 +2,13 @@
 
 function getEventBusName(eventBus) {
   if (eventBus && eventBus.startsWith('arn')) {
-    return eventBus.split('/').pop();
+    const splittedEventBus = eventBus.split('/');
+    // if it's a custom event bus
+    if (splittedEventBus.length === 1) {
+      return splittedEventBus.pop();
+    }
+    // if it's a partner event bus
+    return splittedEventBus.slice(1).join('/');
```

Comment:
This implementation looks a bit weird. Technically it looks as if the goal is to simply strip leading token, if `eventBus` contains `/`, and otherwise pass it through, right?

If it indeed is the intention, wouldn't it be more straightforward to write it as:

```javascript
const separatorIndex = eventBus.indexOf('/'):
return (separateorIndex === -1) ? eventBus : eventBus.slice(separatorIndex + 1); 
```

?

Reply:
Yes, that's the intention. I'll update the PR in the next couple of minutes to include that change.